hack/run-e2e-kind.sh

```diff
@@ -72,6 +72,30 @@ function generate-log {
     kubectl logs deployment/${CLUSTER_NAME}-scheduler -n ${NAMESPACE} > volcano-scheduler.log
 }
 
+function show-log() {
+  if [ -f volcano-admission.log ]; then
```

Comment:
the logic of printing volcano component is similar, whether it's better to do it with a loop.

Reply:
> the logic of printing volcano component is similar, whether it's better to do it with a loop.

good idea